functions

```diff
@@ -877,14 +876,6 @@ install_modules() {
 
     msg "Generating module dependencies"
     install -m644 -t "$moduledest" "$_d_kmoduledir"/modules.{builtin,builtin.modinfo,order}
```

Comment:
Not directly related to this PR, but how do we know which files we need?
The modules directory has: `modules.alias`, `modules.alias.bin`, `modules.builtin`, `modules.builtin.alias.bin`, `modules.builtin.bin`, `modules.builtin.modinfo`, `modules.dep`, `modules.dep.bin`, `modules.devname`, `modules.order`, `modules.symbols`, `modules.symbols.bin`, `modules.softdep`.